meta/main.yml

```diff
@@ -1,11 +1,14 @@
 ---
 dependencies: []
```

Comment:
Should [cisagov/ansible-role-docker](https://github.com/cisagov/ansible-role-docker) be added to the list of dependencies?  It is listed as such in `README.md`.

Reply:
See [here](https://github.com/cisagov/ansible-role-admiral/pull/3#discussion_r1282260148).  I'm requesting one minor change.

Reply:
See [de16765](https://github.com/cisagov/ansible-role-admiral/pull/3/commits/de167652a600793833019eeaf48a54d391831c92).